pyproject.toml

```
@@ -2,54 +2,53 @@
 requires = [
   "setuptools>=62.1",
   "setuptools_scm[toml]>=6.2",
-  "wheel",]
+  "wheel",
+]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "sunkit_magex"
 description = "Solar Magnetic field Extrapolation"
 requires-python = ">=3.10"
 readme = { file = "README.rst", content-type = "text/x-rst" }
-license = { file = "licenses/LICENSE.rst", content-type = "text/plain" }
+license = { file = "licenses/LICENSE.rst" }
 authors = [
+  { name = "David Stansby" },
   { name = "The SunPy Community", email = "sunpy@googlegroups.com" },
```

Not entirely sure this is needed but it should be the other way around. 
```suggestion
  { name = "The SunPy Community", email = "sunpy@googlegroups.com" },
  { name = "David Stansby" },
```